cpp/log/cluster_state_controller-inl.h

```diff
@@ -121,7 +129,7 @@ void ClusterStateController<Logged>::PushLocalNodeState(
 
   const util::Status status(store_->SetClusterNodeState(local_node_state_));
   if (!status.ok()) {
-    LOG(WARNING) << status;
+    LOG(WARNING) << "Couldn't set ClusterNodeState: " << status;
```

Comment:
Good idea.
I've done this one, and we can roll the others over as we spot them.